src/components/CalculatorCard.js

```diff
@@ -7,95 +7,128 @@ import {Accordion, Button, Card, Col, Container, ListGroup, Row} from 'react-boo
-                                            Para achar o Magic Number de um FII, basta preencher o valor atual do FII
-                                            e o valor do último dividendo/provento pago.
-                                        </Card.Body>
-                                    </Accordion.Collapse>
-                                </Card>
-                            </Accordion>
-                        </Col>
-                    </Row>
-                    <Row>
-                        <Col xs={1}></Col>
-                        <Col><p className='text-center'>Nome do Fii</p></Col>
-                        <Col><p className='text-center'>Preço</p></Col>
-                        <Col><p className='text-center'>Dividendos / Proventos</p></Col>
-                        <Col><p className='text-center'>Magic Number</p></Col>
-                    </Row>
-                    <Row>
-                        <ListGroup variant="flush">
-                            {lines.map((item) => {
-                                return (
-                                    <ListGroup.Item key={item.id}>
-                                        <Row>
-                                            <Col xs={1}>
-                                                <Button variant='danger' onClick={() => removeItem(item.id)} className='btn-block' disabled={isDisabled()}><i className="bi bi-x-circle-fill"></i></Button>
-                                            </Col>
-                                            <Col xs={11}>
-                                                {item.element}
-                                            </Col>
-                                        </Row>
-                                    </ListGroup.Item>
-                                )
-                            })}
-                        </ListGroup>
-                    </Row>
-                </Container>
-            </Card.Body>
-            <Card.Footer className='clearfix'>
-                <div className='float-right'>
-                    <Button onClick={addItem}><i className="bi bi-plus-circle-fill"></i> Adicionar Linha</Button>{' '}
-                    <Button disabled><i className="bi bi-cloud-arrow-up-fill"></i> Salvar</Button>
-                </div>
-            </Card.Footer>
-        </Card>
-    )
+  return (
+    <Card>
+      <Card.Header>Descubra o Magic Number dos seus FIIs</Card.Header>
+      <Card.Body>
+        <Container>
+          <Row>
+            <Col xs={12}>
+              <HelpCard />
+            </Col>
+          </Row>
+        <TableHeader />
+        <Row>
+          <ListGroup variant="flush">
+            {lines.map((item) => {
+              return (
+                <CalculatorRow
+                  key={item.id}
+                  id={item.id}
+                  lines={lines}
+                  removeItem={removeItem}
+                />
+              )
+            })}
```

Comment:
```suggestion
            {lines.map(({ id }) => (
               <CalculatorRow
                 key={id}
                 id={id}
                 lines={lines}
                 removeItem={removeItem}
               />
             )
           )}
```

You can return a single component or object from an arrow function if you declare the returned value inside a context block `()` (I'm not how exactly this is called)

Example with object:
```javascript
// also using destructuring and assignment to get properties directly
const getNames = ({ firstName, lastName }) => ({ firstName, lastName });
```